widgets/src/main/java/org/pentaho/gwt/widgets/client/tabs/PentahoTab.java

```diff
@@ -41,6 +41,7 @@ public class PentahoTab extends SimplePanel {
   private Widget content;
   protected Label label = new Label();
   private boolean solutionBrowserShowing;
+  private boolean closeable;
```

Comment:
Please, define a corresponding `public boolean isCloseable()` method (and use that instead elsewhere in this file).

Reply:
Done